src/sensors/SensorObject.ts

```diff
@@ -49,7 +49,7 @@ namespace SensorObject {
       export const from = (value: Value): U8 => {
         switch (value.type) {
           case Type.U8: return value;
-          case Type.Bool: return u8(value.value ? 255 : 0);
+          case Type.Bool: return u8(value.value ? 4095 : 0);
```

Comment:
I think the intent of `U8` was to be an unsigned 8-bit integer (0 - 255), so it might be better to make a separate `U12` (0 - 4095). Or just rename `U8` to `U12` if there aren't any sensors that are 0 - 255.

Reply:
I don't believe there are, to my knowledge every analog sensor is U12.